### PR TITLE
Few fixes to our tutorial sample code

### PR DESCRIPTION
**Pull request by davidiw**
A couple fixes to our tutorials...

1. Gas prices for Tokens are high until we get real maps
2. Rust had some random code :O
3. auth_key api in faucet seems broken -- probably best to just remove it

### CODE REVIEW

**Review by zekun000 — APPROVED** [review]

File: developer-docs-site/static/examples/python/first_transaction.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def generate_transaction(self, sender: str, payload: Dict[str, Any]) -> Dict[str
         txn_request = {
             "sender": f"0x{sender}",
             "sequence_number": str(seq_num),
-            "max_gas_amount": "1000",
+            "max_gas_amount": "2000",
```
Comment (areshand) [inline developer-docs-site/static/examples/python/first_transaction.py:77]: how is max gas amount determined? why does it take high gas amount?

File: developer-docs-site/static/examples/python/first_nft.py [hunk 2]
```diff
@@ -152,8 +152,8 @@ def get_token_id(self, creator: str, collection_name: str, token_name: str) -> i
     print(f"Alice: {alice.address()}")
     print(f"Bob: {bob.address()}")
 
-    faucet_client.fund_account(alice.pub_key(), 10_000_000)
-    faucet_client.fund_account(bob.pub_key(), 10_000_000)
+    faucet_client.fund_account(alice.address(), 10_000_000)
```
Comment (areshand) [inline developer-docs-site/static/examples/python/first_nft.py:155]: seems @CapCap  also has similar changes covered here https://github.com/aptos-labs/aptos-core/pull/372/files#diff-ac78e1760da5c3e43bd9914f36a1415824b276539abdb10dd1c00f9e3dc3e2e0.  we also need to make sure the address has "0x" as prefix.

**Review by CapCap — APPROVED** [review]

**Review by areshand — CHANGES_REQUESTED** [review]

File: developer-docs-site/static/examples/python/first_transaction.py (hunk 1 above)
Comment (davidiw) [reply developer-docs-site/static/examples/python/first_transaction.py:77]: because I ran it and the txn failed because it need 1030 gas

**Pull request merged**